src/components/input/math-input.js

```diff
@@ -753,6 +807,7 @@ class MathInput extends React.Component {
                 ref={node => {
                     this.inputRef = node;
                 }}
+                onKeyUp={this.handleKeyPress}
```

Comment:
Small thing but can you rename `handleKeyPress` to `handleKeyUp` so it maps to the usage? Just so it's easier to connect the definition with the consumption.